modules/st2-auto-form/modules/st2-form-text-field/directive.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+angular.module('main')
+  .directive('st2FormTextField', function () {
+    return {
+      restrict: 'C',
+      require: 'ngModel',
+      scope: {
+        'spec': '=',
+        'options': '=',
+        'ngModel': '=',
+        'disabled': '='
+      },
+      templateUrl: 'modules/st2-auto-form/modules/st2-form-text-field/template.html',
+      link: function (scope, element, attrs, ctrl) {
+        scope.name = ctrl.$name;
+        
+        ctrl.$render = function () {
+          scope.rawResult = ctrl.$viewValue;
+        };
+        
+        scope.$watch('rawResult', function (rawResult) {
+          ctrl.$setViewValue(rawResult);
+        });
```

Comment:
Since we're not doing any kind of transformation here, we can just map scope.ngModel directly to textarea ng-model attribute. Though, you may hold on for a second with that one since I think we may want to make this directive a bit more complex.